### PR TITLE
Closes OFE-100 & OFE-102: Fixed bonding responsive

### PR DESCRIPTION
**Pull request by bbshark99**
https://onomy.atlassian.net/browse/OFE-100
https://onomy.atlassian.net/browse/OFE-102

### CODE REVIEW

File: src/theme/constants.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const responsive = {
   laptop: '1440px',
-  laptopSmall: '1180px',
+  laptopSmall: '1300px',
```
Comment (AndriiBlagodyr) [inline src/theme/constants.js:3]: This change requires retesting of the whole application for tablet/laptopSmall/laptop sizes. @bbshark99  Does all tabs and modals now have a correct view for the mentioned width? I'm asking just to be sure that we didn't break anything with this change. Thanks.

Comment (bbshark99) [reply src/theme/constants.js:3]: @AndriiBlagodyr I think adding new variable might be better. like below:
```
{
  laptopMedium: '1300px',
  laptopSmall: '1180px',
}
```

**Comment by bbshark99** [comment]
@AndriiBlagodyr added `laptop-medium: 1300`

**Review by flppv — CHANGES_REQUESTED** [review]
These tasks can be completed without adding an extra breakpoint. 
As for the footer overlap, need to move it to the bottom;
As for the buy/sell forms need to adjust grid, so items will take 50% of the width.

**Comment by bbshark99** [comment]
@flppv Fixed

**Review by flppv — APPROVED** [review]

**Pull request merged**